internal/uvm/create_wcow.go

```diff
@@ -222,6 +223,23 @@ func prepareConfigDoc(ctx context.Context, uvm *UtilityVM, opts *OptionsWCOW, uv
 		}
 	}
 
+	// Set boot options
+	if opts.DumpDirectoryPath != "" {
+		if info, err := os.Stat(opts.DumpDirectoryPath); err != nil {
+			return nil, fmt.Errorf("failed to stat dump directory %s: %w", opts.DumpDirectoryPath, err)
+		} else if !info.IsDir() {
+			return nil, fmt.Errorf("dump directory path %s isn't a directory", opts.DumpDirectoryPath)
+		}
+		if err := security.GrantVmGroupAccessWithMask(opts.DumpDirectoryPath, security.AccessMaskAll); err != nil {
+			return nil, fmt.Errorf("failed to add SDL to dump directory: %w", err)
+		}
+		debugOpts := &hcsschema.DebugOptions{
+			BugcheckSavedStateFileName:            filepath.Join(opts.DumpDirectoryPath, fmt.Sprintf("%s-savedstate.vmrs", uvm.id)),
+			BugcheckNoCrashdumpSavedStateFileName: filepath.Join(opts.DumpDirectoryPath, fmt.Sprintf("%s-savedstate_nocrashdump.vmrs", uvm.id)),
```

Comment:
What is the difference between these two settings? Can you add a comment?

Reply:
Added comments in the schema file.